src/main/java/util/Utility.java

```diff
@@ -72,7 +72,10 @@ public static boolean writeFile(String fileName, String content, int issueCount)
                 writer.println(content);
                 writer.close();
 
-                long sizeAfterWrite = Files.size(Paths.get(fileName));
+                long sizeAfterWrite = 0;
+                if (issueCount > 0) {
```

Comment:
Thank you for noticing that. I put the check because I altered `testJsonExplosionDetection` so that when `issueCount` is 0, `corruptedJson` will be `false`. It was always corrupted because the test case of 0 `issueCount` does not create the file. I will look at it again.